ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.tsx

```diff
@@ -23,11 +23,7 @@ import {
 import { getDate, getDateTime, getDistanceStrictAsPhrase } from 'utils/dateUtils';
 import useModal from 'hooks/useModal';
 
-export type RequestContext =
-    | 'PENDING_REQUESTS'
-    | 'APPROVED_DEFERRALS'
-    | 'APPROVED_FALSE_POSITIVES'
-    | 'DENIED_REQUESTS';
+export type RequestContext = 'NORMAL' | 'LATEST';
```

Comment:
I've revised the context type usage. Previously, we used the context value to decide the data subset for rendering views. In the `Pending Request`s tab, it showed updated request data, while elsewhere, it displayed non-updated data (from `deferralRequest` or `falsePositiveRequest` fields). I've simplified this, as the other context values seemed unnecessary, particularly since we need to display both requested update data and latest approved data in the request details view. This approach feels more coherent, though I'm still not sure if these terms are the best. Open to other ideas for the terms

Reply:
Yeah, it is. I think your suggestion fits better in this case. I'll try to make that change in this PR